src/FbTk/App.cc

```diff
@@ -132,7 +132,9 @@ bool App::setenv(const char* key, const char* value) {
     }
 
     // create the new environment
-    strncpy(newenv, key, key_size);
+    strncpy(newenv, key, key_size+1); // +1 avoids warning from very clever
```

Comment:
this is an unrelated change to the topic.

not unwanted, but unrelated. and thus it belongs it its own commit